assets/xml/objects/object_gj.xml

```diff
@@ -0,0 +1,23 @@
+<Root>
+    <File Name="object_gj" Segment="6">
+        <DList Name="gGjDL_000DC0" Offset="0xDC0"/>
+        <Collision Name="gGjCol_001B70" Offset="0x1B70"/>
+        <DList Name="gGjDL_001D20" Offset="0x1D20"/>
+        <Collision Name="gGjCol_001F70" Offset="0x1F70"/>
+        <DList Name="gGjDL_002160" Offset="0x2160"/>
+        <Collision Name="gGjCol_002448" Offset="0x2448"/>
+        <DList Name="gGjDL_002600" Offset="0x2600"/>
+        <Collision Name="gGjCol_002850" Offset="0x2850"/>
+        <DList Name="gGjDL_002A40" Offset="0x2A40"/>
+        <Collision Name="gGjCol_002D28" Offset="0x2D28"/>
+        <DList Name="gGjDL_002E80" Offset="0x2E80"/>
+        <Collision Name="gGjCol_002FE4" Offset="0x2FE4"/>
+        <DList Name="gGjDL_003190" Offset="0x3190"/>
+        <Collision Name="gGjCol_0033E0" Offset="0x33E0"/>
+        <DList Name="gGjDL_003710" Offset="0x3710"/>
+        <Collision Name="gGjCol_003AF0" Offset="0x3AF0"/>
+
+        <!-- I don't know what is this -->
+        <Blob Name="gGjBlob_008B20" Size="0x1C00" Offset="0x8B20"/>
```

Comment:
Just checked and I can tell that there are some textures in this region, I'd try adding those to the xml and seeing what unaccounted data remains, there might also be cutscene data in this too, though I don't think the game ever does that.

Reply:
looking at texture explorer, my best guesses are this:
        <Texture Name="gGjTex_008B20" OutName="g_GjTex_008B20" Format="rgba32" Width="32" Height="32" Offset="0x8B20"/>
        <Texture Name="gGjTex_009B20" OutName="g_GjTex_009B20" Format="rgb5a1" Width="32" Height="32" Offset="0x9B20"/>
        <Texture Name="gGjTex_00A320" OutName="g_GjTex_00A320" Format="rgba32" Width="16" Height="16" Offset="0xA320"/>

Reply:
Are you sure they are textures? The first and second ones look like garbage data. The third one may be an unused ground texture or something.

I added this region as a blob because I gave up trying to figure it out what could it be, like, I see those textures and I think "this isn't right"